template.distillery/PROJECT_NAME_handlers.eliomi

```diff
@@ -0,0 +1,59 @@
+(* This file was generated by Ocsigen-start.
+   Feel free to use it, modify it, and redistribute it as you wish. *)
+
```

Comment:
I also added some comments to separate the different handlers because it was confusing to see no separation between the handlers definitions.